app/views/notes/index.html.erb

```diff
@@ -26,6 +26,10 @@
 <% else %>
   <%= render :partial => "notes_paging_nav" %>
 
+  <svg width="0" height="0" class="end-100 position-absolute">
```

Comment:
As all three places that include the partial wrap it in the same `svg` element why not include that in the partial?